src/parts/Form/index.jsx

```diff
@@ -63,16 +64,11 @@ const Form = () => {
 
     // NOTE • Refs are needed to access state from within events.onmessage callbacks
     const statsRef = useRef(stats);
-    const nodeRef = useRef(node);
 
     useEffect(() => {
         statsRef.current = stats;
     }, [stats]);
 
-    useEffect(() => {
```

Comment:
as we discussed async, I think nodeRef should be restored

Reply:
I think I thought of another way - passing `node` from context to `startNode` that's being called in [this useEffect](https://github.com/eigerco/lumina-front/pull/55/files#diff-612d149cf44513fa470680443f89faf2b3f9604e61ba936a83b75a1ee2cd7fddR305). As I understand it it should result in updated node instance getting passed. lmk what you think.